backtrace-library/src/main/java/backtraceio/library/BacktraceClient.java

```diff
@@ -24,6 +27,21 @@
      */
     private BacktraceANRWatchdog anrWatchdog;
 
+    /**
+     * Backtrace Battery Helper
+     */
+     private BatteryStateHelper batteryStateHelper;
```

Comment:
Maybe the interface should be called `BatteryStateHelper` and the implementation called `BacktraceBatteryStateHelper`. This is just a nit though.